### PR TITLE
add vhs to arkade get

### PR DESCRIPTION
**Pull request by NikhilSharmaWe**
## Description

This PR adds vhs to arkade get

## Motivation and Context

- [x] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))
Fixes: #927  

## How Has This Been Tested?

Running locally `arkade get vhs`.

If updating or adding a new CLI to `arkade get`, run:

```
> go build && ./hack/test-tool.sh vhs
+ ./arkade get vhs --arch arm64 --os darwin --quiet
+ file /home/nikhilsharmawe/.arkade/bin/vhs
/home/nikhilsharmawe/.arkade/bin/vhs: Mach-O 64-bit arm64 executable, flags:<|DYLDLINK|PIE>
+ rm /home/nikhilsharmawe/.arkade/bin/vhs
+ echo

+ ./arkade get vhs --arch x86_64 --os darwin --quiet
+ file /home/nikhilsharmawe/.arkade/bin/vhs
/home/nikhilsharmawe/.arkade/bin/vhs: Mach-O 64-bit x86_64 executable
+ rm /home/nikhilsharmawe/.arkade/bin/vhs
+ echo

+ ./arkade get vhs --arch x86_64 --os linux --quiet
+ file /home/nikhilsharmawe/.arkade/bin/vhs
/home/nikhilsharmawe/.arkade/bin/vhs: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=uEpPGWWW3vVrEq1Z00Uk/-q4NEqjZA3iKhY__sy9j/f1nk3H9zrNh8v8QsmDsV/DgzZqprnlH9_70-WQRto, stripped
+ rm /home/nikhilsharmawe/.arkade/bin/vhs
+ echo

+ ./arkade get vhs --arch aarch64 --os linux --quiet
+ file /home/nikhilsharmawe/.arkade/bin/vhs
/home/nikhilsharmawe/.arkade/bin/vhs: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, Go BuildID=8CIwyELnf_nP3EHm9R3u/izScT2p5SrKtBkCYeCoW/JcPj3SV8QUNWCrlhFDxL/ENMl1m_p7qC_sWzVU2Mi, stripped
+ rm /home/nikhilsharmawe/.arkade/bin/vhs
+ echo

+ ./arkade get vhs --arch x86_64 --os mingw --quiet
+ file /home/nikhilsharmawe/.arkade/bin/vhs.exe
/home/nikhilsharmawe/.arkade/bin/vhs.exe: PE32+ executable (console) x86-64 (stripped to external PDB), for MS Windows
+ rm /home/nikhilsharmawe/.arkade/bin/vhs.exe
+ echo
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [x] I have updated the list of tools in README.md if (required) with `./arkade get --format markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

**Comment by alexellis** [comment]
@NikhilSharmaWe 

> If updating or adding a new CLI to arkade get, run:

We need that output please.

Alex

File: pkg/get/tools.go [hunk 1]
```diff
@@ -3687,5 +3687,30 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
...
+					{{$arch := .Arch}}
+					{{ if (eq .Arch "x86_64") -}}
+					{{$arch = "x86_64"}}
+					{{- else if eq .Arch "aarch64" -}}
+					{{$arch = "arm64"}}
+					{{- end -}}
+
+					{{$osStr := ""}}
+					{{- if eq .OS "darwin" -}}
+					https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}_Darwin_{{$arch}}.tar.gz
```
Comment (alexellis) [inline pkg/get/tools.go:3706]: I think you may be able to omit the prefix of https://github.com/{{.Owner}}/{{.Repo}}/releases/download/

Or just have it on the last line so it's not duplicated.

Please look at other similar CLIs.

**Review by alexellis — CHANGES_REQUESTED** [review]
Please see comment on repetition of URL

And lack of testing tool output

File: pkg/get/tools.go (hunk 1 above)
Comment (NikhilSharmaWe) [reply pkg/get/tools.go]: I have updated, PTAL.
The URL format is different in the case of Darwin and Linux/Windows therefore need 2 separate cases.

Please inform if any issues.

**Comment by NikhilSharmaWe** [comment]
@alexellis Could you please review?

**Comment by alexellis** [comment]
We still need the output from `go build && ./hack/test-tool.sh TOOL_NAME` in the PR description please

And the PR will need a rebase - https://www.youtube.com/watch?v=8j0H6urZ-bU

**Comment by NikhilSharmaWe** [comment]
@alexellis Rebased the PR and updated the PR description.

**Pull request merged**